src/formBuilder/superfluidPaymentInput.jsx

```diff
@@ -0,0 +1,47 @@
+import React, { useEffect } from 'react';
+import { useParams } from 'react-router';
+import { SF_ACTIVE_STREAMS } from '../graphQL/superfluid-queries';
+import { graphQuery } from '../utils/apollo';
```

Comment:
move graphQuery and supportedChains below Payment Input and should only have one space after graphQuery